server-connect-base/Dockerfile.ubi8

```diff
@@ -50,6 +50,8 @@ RUN echo "===> Installing ${COMPONENT}..." \
     && echo "===> Adding confluent repository...${CONFLUENT_PACKAGES_REPO}" \
     && rpm --import ${CONFLUENT_PACKAGES_REPO}/archive.key \
     && printf "[Confluent.dist] \n\
+    && rm -rf /usr/share/java/kafka/grpc-*.jar
+    && rm -rf /usr/share/java/confluent-metadata-service/grpc-*.jar
```

Comment:
@akanimesh7 why not taking a safer approach of investigating why `grpc-context-1.45.1.jar` is end up there, so that we can only exclude one jar.

This change is dangerous because it will remove every grpc jars.